pkg/bmc/irmc.go

```diff
@@ -74,7 +74,7 @@ func (a *iRMCAccessDetails) ManagementInterface() string {
 }
 
 func (a *iRMCAccessDetails) PowerInterface() string {
-	return ""
+	return "ipmitool"
```

Comment:
We don't populate ipmi credentials above, will it work?

Reply:
In my test environment, ipmi related information can be successfully obtained.
```
| driver_info            | {'deploy_kernel': 'http://localhost:6181/images/ironic-python-agent.kernel',                |
|                        | 'deploy_ramdisk': 'http://localhost:6181/images/ironic-python-agent.initramfs',             |
|                        | 'irmc_address': '192.168.30.25', 'irmc_password': '******', 'irmc_username': 'admin',       |
|                        | 'ipmi_address': '192.168.30.25', 'ipmi_username': 'admin', 'ipmi_password': '******'}   
```

Reply:
After I manually changed the value of node's `power_interface` to `ipmitool`, `sofftshutdown` can be executed correctly.
@dtantsur So do you have any questions about this modification?

Reply:
Looks like the iRMC driver takes care of it: https://opendev.org/openstack/ironic/src/branch/master/ironic/drivers/modules/irmc/common.py#L160-L174